### PR TITLE
add comment in hack/update-*.sh

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

kubernetes/hack/update-all.sh
kubernetes/hack/update-bazel.sh
kubernetes/hack/update-codegen.sh
kubernetes/hack/update-generated-api-compatibility-data.sh
kubernetes/hack/update-generated-device-plugin-dockerized.sh
kubernetes/hack/update-generated-device-plugin.sh
kubernetes/hack/update-generated-docs.sh
kubernetes/hack/update-generated-kms-dockerized.sh
kubernetes/hack/update-generated-kms.sh
kubernetes/hack/update-generated-kubelet-plugin-registration-dockerized.sh
kubernetes/hack/update-generated-kubelet-plugin-registration.sh
kubernetes/hack/update-generated-pod-resources-dockerized.sh
kubernetes/hack/update-generated-pod-resources.sh
kubernetes/hack/update-generated-protobuf-dockerized.sh
kubernetes/hack/update-generated-protobuf.sh
kubernetes/hack/update-generated-runtime-dockerized.sh
kubernetes/hack/update-generated-runtime.sh
kubernetes/hack/update-generated-swagger-docs.sh
kubernetes/hack/update-gofmt.sh
kubernetes/hack/update-import-aliases.sh
kubernetes/hack/update-openapi-spec.sh
kubernetes/hack/update-translations.sh
kubernetes/hack/update-vendor-licenses.sh
kubernetes/hack/update-vendor.sh
kubernetes/hack/update-workspace-mirror.sh

update the comment for them

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/86597

**Special notes for your reviewer**:
/cc @fejta 

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

File: hack/update-bazel.sh [hunk 1]
```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This script updates the bazel compilation files once. We should run 
+# `hack/update-bazel.sh` once we modify the reference package of the Go file.
+# Usage: `hack/update-bazel.sh`.
```
Comment (fejta) [inline hack/update-bazel.sh]: These seem pretty pointless

Comment (tanjunchen) [reply hack/update-bazel.sh]: done . i will update it

**Comment by yuxiaobo96** [comment]
/lgtm
good job !

**Pull request merged**